src/vm/prestub.cpp

```diff
@@ -369,12 +369,17 @@ PCODE MethodDesc::PrepareILBasedCode(PrepareCodeConfig* pConfig)
     {
         pCode = GetPrecompiledCode(pConfig);
     }
+
     if (pCode == NULL)
     {
         LOG((LF_CLASSLOADER, LL_INFO1000000,
             "    In PrepareILBasedCode, calling JitCompileCode\n"));
         pCode = JitCompileCode(pConfig);
     }
+    else
+    {
+        DACNotifyCompilationFinished(this, pCode);
```

Comment:
Hmm, this isn't intended to be necessary although I don't mind putting this in with a comment and leaving the issue #22265 open.

What I would expect to happen if things were working smoothly is that the debugger would bind the breakpoint in response to a module load at roughly this callstack:
    
    IssueDebuggerBPCommand ( CLRDATA_ADDRESS addr )
    Breakpoints::ResolveMethodInstances(IXCLRDataMethodDefinition *pMeth, DWORD ilOffset)
    Breakpoints::ResolvePendingBreakpoint(TADDR addr, PendingBreakpoint *pCur)
    Breakpoints::Update(TADDR mod, BOOL isNewModule)

Triggering an event for each method as it loads also works, but has higher performance overhead vs. doing a single sweep at module load.